content/en/docs/reference/glossary/service.md

```diff
@@ -11,8 +11,8 @@ tags:
 - fundamental
 - core-object
 ---
-An abstract way to expose an application running on a set of {{< glossary_tooltip text="Pods" term_id="pod" >}} as a network service.
+A method for exposing an application service that is running as one or more {{< glossary_tooltip text="Pods" term_id="pod" >}} in your cluster, using IP networking.
```

Comment:
Since we're defining service, we should consider omitting the word in this line. It could cause confusion.
```suggestion
A method for exposing an application that is running as one or more {{< glossary_tooltip text="Pods" term_id="pod" >}} in your cluster, using IP networking.
```

Reply:
I agree, but can we use "service" with "interface" or similar?

Reply:
Perhaps you can suggest a new wording @geoffcline ? I don't think I can imagine what you're proposing.